### PR TITLE
Skip BinSkim scan for msalruntime*.dll

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
- Skip BinSkim scan for msalruntime*.dll by adding a scan list.
- Remove pdb files and their helper scripts

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: .azure-pipelines/security-tools.yml [hunk 1]
```diff
@@ -53,21 +53,21 @@ jobs:
...
 
   - task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@3
     displayName: Run BinSkim
     inputs:
-      InputType: 'Basic'
-      Function: 'analyze'
-      AnalyzeTarget: '$(Build.SourcesDirectory)/artifacts/Debug/*.dll;$(Build.SourcesDirectory)/artifacts/*.exe'
-      AnalyzeStatistics: true
+      InputType: 'CommandLine'
+      arguments: 'analyze @$(Build.SourcesDirectory)/artifacts/MyFileList.rsp --recurse'
```
Comment (BethanyZhou) [inline .azure-pipelines/security-tools.yml:70]:
```suggestion
      arguments: 'analyze @$(Build.SourcesDirectory)/artifacts/MyFileList.rsp --config default --recurse --statistics'
```

File: .azure-pipelines/security-tools.yml [hunk 2]
```diff
@@ -53,28 +53,21 @@ jobs:
...
-      pwsh: true
   
   - task: PowerShell@2
     displayName: Generate a response text file for BinSkim
     inputs:
       targetType: 'inline'
       script: |
-        New-Item $(Build.SourcesDirectory)/artifacts/MyFileList.rsp -ItemType File -Force
-        (Get-ChildItem -Path .\artifacts\Debug -Include *.dll,*.exe -Exclude msalruntime.dll,msalruntime_arm64.dll,msalruntime_x86.dll -Recurse).FullName > $(Build.SourcesDirectory)/artifacts/MyFileList.rsp
+        New-Item $(Build.SourcesDirectory)/artifacts/ScannedFileList.rsp -ItemType File -Force
```
Comment (isra-fel) [inline .azure-pipelines/security-tools.yml:62]: ScannedFileList gives you an impression that files on this list have been scanned, but that's the opposite. Use FilesToScan

Comment (BethanyZhou) [inline .azure-pipelines/security-tools.yml:62]:
```suggestion
        New-Item $(Build.SourcesDirectory)/artifacts/FilesToScan.rsp -ItemType File -Force
```

File: .azure-pipelines/security-tools.yml [hunk 3]
```diff
@@ -53,28 +53,21 @@ jobs:
...
+        (Get-ChildItem -Path .\artifacts\Debug -Include *.dll,*.exe -Exclude msalruntime.dll,msalruntime_arm64.dll,msalruntime_x86.dll -Recurse).FullName > $(Build.SourcesDirectory)/artifacts/ScannedFileList.rsp
```
Comment (BethanyZhou) [inline .azure-pipelines/security-tools.yml:63]:
```suggestion
        (Get-ChildItem -Path .\artifacts\Debug -Include *.dll,*.exe -Exclude msalruntime.dll,msalruntime_arm64.dll,msalruntime_x86.dll -Recurse).FullName > $(Build.SourcesDirectory)/artifacts/FilesToScan.rsp
```

**Review by isra-fel — APPROVED** [review]

**Pull request merged**